src/Text/Text.tsx

```diff
@@ -1,33 +1,28 @@
 import React, { FC, ReactNode } from 'react';
 import { styled } from '@storybook/theming';
 import {
+  breakpoint,
   color as tokenColor,
-  fontFamily as tokenFontFamily,
-  fontSize as tokenFontSize,
   fontWeight as tokenFontWeight,
-  lineHeight as tokenLineHeight,
 } from '../_tokens';
-import { minSm } from '../_helpers';
+import { typography } from '../_helpers';
+
+type Breakpoints = keyof typeof breakpoint;
+
+export function objectKeys<Obj, Keys extends (keyof Obj)[]>(obj: Obj): Keys {
+  return Object.keys(obj as any) as Keys;
+}
 
 export interface TextProps {
   children: ReactNode;
   variant?:
-    | 'heading4xl'
```

Comment:
Just want to confirm that these old variants are still available? They're being used on the Figma and case study pages.

Reply:
So my plan was to use this PR to test this new component without these variants on Chromatic. I'll use the canary version to test on Chromatic to update all versions on Chromatic and remove these variables. When everything is ok, I'll push Tetra, update the version on Chromatic and push a PR on Chromatic.

Reply:
That would be great. I'm pretty sure it's used in a few places in the chromatic repo, you'll likely have to adjust those.

Reply:
I saw the bug thanks to Chromatic by the way. :) So glad this is in place.